### DISCUSSION TITLE
Only deploy part of the monorepo on merge

### DISCUSSION THREAD

**Discussion by leo-petrucci** [vercel/turborepo#4099] [Help] [answered]
I'm currently trying to rework 10/15 separate microservices repos into a single monorepo. Local development works great, but I'm wondering how deploying each of these microservices will work.

Previously I was able to deploy each repo to production if any branch was merged to each repo's main branch. However with a monorepo setup I have no way of telling which specific microservice was modified and just deploy that one.

Is there a smart way to do this that I'm missing?

**Comment by leo-petrucci** [comment, marked as answer]
Nevermind, I just found out Github supports `paths` in actions to define which subfolders get changed.

I think this might be useful for the docs maybe?

**Reply by mehulkar** [reply]
`npx turbo-ignore` may also work for you https://turbo.build/repo/docs/core-concepts/monorepos/skipping-tasks

**Comment by mehulkar** [comment]
We generally point people to `npx turbo-ignore`: https://turbo.build/repo/docs/core-concepts/monorepos/skipping-tasks
